src/main/java/hudson/plugins/sonar/client/SQProjectResolver.java

```diff
@@ -43,38 +41,35 @@ public SQProjectResolver(HttpClient client) {
    * Errors that should be displayed are included in {@link ProjectInformation#getErrors()}.
    */
   @CheckForNull
-  public ProjectInformation resolve(@Nullable String projectUrl, @Nullable String ceTaskId, String installationName) {
+  public ProjectInformation resolve(@Nullable String serverUrl, @Nullable String projectDashboardUrl, String ceTaskId, String installationName) {
     SonarInstallation inst = SonarInstallation.get(installationName);
     if (inst == null) {
       Logger.LOG.info(() -> "Invalid installation name: " + installationName);
       return null;
     }
+    if (serverUrl == null) {
+      Logger.LOG.info(() -> String.format(Locale.US, "No server url."));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: String contains no format specifiers. ([squid:S3457](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS3457))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.jenkins-ci.plugins%3Asonar&pullRequest=90&issues=AWIwh_yk_wH-bYHl3dHv&open=AWIwh_yk_wH-bYHl3dHv)

[ISSUE_KEY]: AWIwh_yk_wH-bYHl3dHv